src/components/general/Footer.js

```diff
@@ -3,7 +3,22 @@ import styled from "styled-components";
 
 const Footer = () => (
   <Container>
+    <LeftSection>
+      <a href="imprint">Imprint</a>
+    </LeftSection>
```

Comment:
What is imprint? And should it be a Link instead of an a tag?

Reply:
Imprint contains some information about who's responsible for the website. It's legally required for any website in Germany. I will change it to a link and will add the actual page.